packages/node_modules/@ciscospark/internal-plugin-presence/src/presence.js

```diff
@@ -0,0 +1,145 @@
+/**!
+ *
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+import '@ciscospark/internal-plugin-wdm';
+
+import {SparkPlugin} from '@ciscospark/spark-core';
+import {transformSingleStatusResponse, transformStatusResponses} from './transforms';
+
+const defaultSubscriptionTtl = 600;
+
+/**
+ * @class
+ * @extends SparkPlugin
+ */
+const Presence = SparkPlugin.extend({
+  namespace: `Presence`,
+
+  /**
+   * Gets the current presence status of a given person id
+   * @param {string} personId
+   * @returns {Promise}
+   */
+  get(personId) {
+    if (!personId) {
+      return Promise.reject(new Error(`A person id is required`));
+    }
+    return this.postCompositions([personId]);
```

Comment:
You can do a GET for a single user, but the response is completely different, so I just used the bulk endpoint for one or many